java/apirag-core/src/main/java/com/datasqrl/ai/models/ResponseMessage.java

```diff
@@ -0,0 +1,18 @@
+package com.datasqrl.ai.models;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class ResponseMessage {
```

Comment:
As far as I understood the code so far, the ResponseMessage is a class made for the SimpleServer test application, since I interpret it as an ApiModel. Does it make sense to pull it into the core?

Reply:
The ResponseMessage is only used by SimpleServer to communicate with the frontend.